src/main/java/core/selenium/webdrivers/BrowserFactory.java

```diff
@@ -28,6 +28,12 @@
         put(BrowserType.FIREFOX, new FireFox());
     }};
 
+    /**
+     * Contructor.
+     */
+    public BrowserFactory() {
+    }
+
     /**
      * Method to let set a type of browser and return his driver.
```

Comment:
please read again the documentation, must be written in third person

Reply:
I think is not necessary comment that because the constructor is by default and it is empty.